cilium-docker-plugin.Dockerfile

```
@@ -1,6 +1,6 @@
 # (first line comment needed for DOCKER_BUILDKIT use)
 #
-FROM docker.io/library/golang:1.14.7 as builder
+FROM docker.io/library/golang:1.15 as builder
```

```suggestion
FROM docker.io/library/golang:1.15.0 as builder
```
I think that for the Go Docker image, the `.0` needs to be appended. If not. when Go 1.15.1 is released it'll be automatically picked up which is probably not what we want.

---

Good point, thanks. Since this was updated automatically based on `GO_VERSION` using `make update-golang` I'll have to adjust these manually. We want to keep `1.15` where we don't refer to `docker.io/library/golang`.